concreate/descriptor.py

```diff
@@ -96,6 +97,15 @@ def _process_execute(self):
             if 'user' not in execute:
                 execute['user'] = DEFAULT_USER
 
+    def _process_run(self):
+        """ Make sure the user is set for cmd/entrypoint  """
+
+        if 'run' not in self.descriptor:
+            self.descriptor['run'] = {}
+
+        if not self.descriptor['run'].get('user', None):
```

Comment:
cannot it be:
'if' user not in self.descriptor['run']:
Also default=None in .get() for dictionary - so you don't need to specify it.

Reply:
Sure, it can. Code looked earlier a bit differently and I didn't catch this. Thanks!